efk/README.md

```diff
@@ -0,0 +1,33 @@
+# EFK Stack
+
+EFK stack sets up log storage and analytics stack using Elasticsearch, Fluentd and Kibana.
+
+Currently Kontena EFK setup is actually using multiple stacks to make all the parts highly re-usable.
+
+Following chapters describe how to set up EFK using ready made stacks. It's advised to install the stacks in the order defined here.
+
+## Elasticsearch
+
+Set up Elasticsearch using `kontena/elasticsearch` stack. It needs volumes to store Elasticsearch data persistently. Those volumes should be `instance` scoped as Elasticsearch itself replicates the data.
```

Comment:
As a user it would be nice to have link to these stacks:
[`kontena/elasticsearch`](https://github.com/kontena/kontena-stacks/elasticsearch)